tests/unit/EligibleAssetRemoved.test.ts

```diff
@@ -0,0 +1,86 @@
+import { Address, BigInt } from '@graphprotocol/graph-ts';
+import {
+    afterEach,
+    assert,
+    beforeEach,
+    clearStore,
+    describe,
+    test,
+} from 'matchstick-as/assembly/index';
+import { App, VoucherType } from '../../generated/schema';
+import { handleEligibleAssetRemoved } from '../../src/voucherHub';
+import { createEligibleAssetRemovedEvent } from './utils/utils';
+
+describe('EligibleAssetRemovedEvent', () => {
+    beforeEach(() => {
+        clearStore();
+    });
+
+    afterEach(() => {
+        clearStore();
+    });
+
+    test('Should update voucherType when the entity exists', () => {
```

Comment:
I suggest to change the name of the test to smth like "should remove" instead of should update

Reply:
fix